go.mod

```diff
@@ -2,116 +2,115 @@ module github.com/rancher/aks-operator
 
 go 1.20
 
-replace k8s.io/client-go => k8s.io/client-go v0.25.4
+replace (
+	github.com/rancher/wrangler => github.com/rancher/wrangler v1.1.1-0.20230831050635-df1bd5aae9df
+	k8s.io/client-go => k8s.io/client-go v0.27.5
+)
 
 require (
 	github.com/Azure/azure-sdk-for-go v55.7.0+incompatible
 	github.com/Azure/go-autorest/autorest v0.11.29
 	github.com/Azure/go-autorest/autorest/adal v0.9.23
 	github.com/Azure/go-autorest/autorest/to v0.4.1-0.20210111195520-9fc88b15294e
+	github.com/drone/envsubst/v2 v2.0.0-20210730161058-179042472c46
+	github.com/golang/mock v1.6.0
 	github.com/onsi/ginkgo/v2 v2.11.0
 	github.com/onsi/gomega v1.27.10
-	github.com/rancher/lasso v0.0.0-20221227210133-6ea88ca2fbcc
-	github.com/rancher/machine v0.15.0-rancher102
-	github.com/rancher/wrangler v1.1.0
+	github.com/pkg/errors v0.9.1
+	github.com/rancher-sandbox/ele-testhelpers v0.0.0-20221213084338-a8ffdd2b87e3
+	github.com/rancher/lasso v0.0.0-20230830164424-d684fdeb6f29
+	github.com/rancher/machine v0.15.0-rancher103
+	github.com/rancher/rancher/pkg/apis v0.0.0-20230915232223-a9ea4ce4a5ba
```

Comment:
only bumped `github.com/rancher/rancher/pkg/apis` to latest after reviews as per https://github.com/rancher/eks-operator/issues/241#issuecomment-1723304785